kubetest/e2e.go

```diff
@@ -587,7 +587,7 @@ func nodeTest(nodeArgs []string, testArgs, nodeTestArgs, project, zone string) e
 	return control.FinishRunning(exec.Command("go", runner...))
 }
 
-func kubemarkTest(testArgs []string, dump, numNodes string) error {
+func kubemarkTest(testArgs []string, dump, numNodes string, deploy deployer) error {
```

Comment:
Could you explain what's the reason to pass a deployer here? Is it needed?

Reply:
to make the deploy.IsUp call above, basically it's doing `./hack/e2e-internal/e2e-status.sh`